src/tests/Microsoft.PowerFx.Core.Tests.Shared/IntellisenseTests/SignatureHelpTest.cs

```diff
@@ -24,13 +24,22 @@ public class SignatureHelpTest : IntellisenseTestBase
         /// instead include the subpath bin/(Debug|Release).AnyCPU, depending on whether the assembly was
         /// built in debug or release mode.
         /// </summary>
-        private static readonly string _baseDirectory = Path.Join(Directory.GetCurrentDirectory(), "IntellisenseTests", "TestSignatures");
 
-        private static readonly string _signatureHelpDirectory = RegenerateSignatureHelp ?
-            _baseDirectory
-                .Replace(Path.Join("bin", "Debug", "netcoreapp3.1"), string.Empty)
-                .Replace(Path.Join("bin", "Release", "netcoreapp3.1"), string.Empty) :
-            _baseDirectory;
+        private static readonly string _baseDirectory = PathJoin(Directory.GetCurrentDirectory(), "IntellisenseTests", "TestSignatures");
+
+        private static readonly string _signatureHelpDirectory = RegenerateSignatureHelp
+            ? _baseDirectory
+                .Replace(PathJoin("bin", "Debug", FrameworkVersion()), string.Empty)
+                .Replace(PathJoin("bin", "Release", FrameworkVersion()), string.Empty)
+                .Replace(PathJoin("bin", "DebugAll", FrameworkVersion()), string.Empty)
+                .Replace(PathJoin("bin", "ReleaseAll", FrameworkVersion()), string.Empty)
+                .Replace(PathJoin("bin", "Debug462", FrameworkVersion()), string.Empty)
+                .Replace(PathJoin("bin", "Release462", FrameworkVersion()), string.Empty)
+                .Replace(PathJoin("bin", "Debug80", FrameworkVersion()), string.Empty)
+                .Replace(PathJoin("bin", "Release80", FrameworkVersion()), string.Empty)
+                .Replace(PathJoin("bin", "Debug80", FrameworkVersion()), string.Empty)
+                .Replace(PathJoin("bin", "Release80", FrameworkVersion()), string.Empty)
+            : _baseDirectory;
```

Comment:
I notice similar code in FormulaTypeSerializerSnapshotTests - can we share with some helper? (put in shims) 

This would be hard to get right elsewise.

Reply:
Will be done in a future PR